pages/help/help_domain-management.md

```diff
@@ -15,71 +15,113 @@ Manage DNS information, domain contacts, and more. Please keep information about
 - [Update your contact information (email, phone)](#update-your-contact-information-(email%2C-phone))
 - [Add or update the security email for your .gov domain](#add-or-update-the-security-email-for-your-.gov-domain)
-- [Add or update contacts to help manage your domain](#add-or-update-contacts-to-help-manage-your-domain)
+- [Add a domain manager to your .gov domain](#add-a-domain-manager-to-your-.gov-domain)
 
 
-## Manage DNS information for your domain
+## Manage DNS name servers
 
-A name server is a server within the Domain Name System (DNS) that answers queries about a domain name. DNS is the internet service that translates your domain name into an IP address. This makes it possible for people to access your online service by using a name instead of a numbered address. Before your approved .gov domain can be used, you’ll need to connect it to your DNS hosting service. You’ll also need to enter your name server information on the .gov domain management site. Follow the steps below to do that.
+The Domain Name System (DNS) is the internet service that translates your domain name into an IP address. This makes it possible for people to access your online service by using a name instead of a numbered address.
 
-1. [Sign in with your Login.gov account](#)
+Before your approved .gov domain can be used, [you’ll first need to connect it to a DNS hosting service](https://beta.get.gov/domains/moving/#find-dns-hosting-services). After you’ve set up hosting, you’ll need to enter your name server information on the .gov registrar. Name server records indicate which DNS server is authoritative for your domain. You must add at least two name servers (13 max).
+
+1. Sign in to the [.gov registrar](https://manage.get.gov){.usa-link--external}. using your Login.gov account.
 2. Click the “Manage” link for your .gov domain.
-3. Click “DNS name servers” on the left-side navigation.
-4. Add your name server information.
+3. Click “Name servers” (under “DNS”) on the left-side navigation.
+4. Enter your name servers in the required format using only supported characters (letters [A-Z], numbers [0-9], the minus sign [-], and the period [.]).
 5. Click “Save.”
 
+If you self-host your DNS or use custom addresses for your name server, you will need to add IP addresses for each name server in either the IPv4 or IPv6 format (multiple IP addresses must be separated with commas).
+
+## Enable DNSSEC on your domain
+
+DNSSEC, or DNS Security Extensions, is an additional security layer to protect your website. Enabling DNSSEC ensures that when someone visits your domain, they can be certain that it’s connecting to the correct server, preventing potential hijacking or tampering with your domain's records. 
+
+<strong>Note: It is strongly recommended that you only enable DNSSEC if you know how to set it up properly at your hosting service. If you make a mistake, it could cause your domain name to stop working.</strong>
```

Comment:
```suggestion
<strong>Note: It is strongly recommended that you only enable DNSSEC if you know how to set it up properly with your hosting service. If you make a mistake, it could cause your domain name to stop working.</strong>
```
Minor typo